### PR TITLE
Add a CODEOWNERS file

### PR DESCRIPTION
**Pull request by smortex**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (smortex) [inline .github/CODEOWNERS:1]: @alltilla are you okay to be added as a code owner?

This is mainly to get review requests assigned automatically for PRs from Dependabot.  My "standard policy" on this kind of repo is that I like to have a 3rd party approval before merging, and after a few days, if nobody complained, it's fine to merge changes.

Comment (alltilla) [reply .github/CODEOWNERS:1]: Yes, thank you.

**Review by alltilla — APPROVED** [review]

**Pull request merged**
